upgrade.yml

```diff
@@ -422,6 +422,13 @@
     # FIXME Added since we are changing swift init scripts in 3.0.x
     # This will stop services started by "old" scripts. This can be removed
     # after all systems have migrated to 3.0.x
+    - name: set swiftops user as owner of ring definition
+      file:
+        path: /etc/swift/ring_definition.yml
+        owner: swiftops
+        group: swiftops
+      run_once: true
+      delegate_to: "{{ groups['swiftnode_primary'][0] }}"
```

Comment:
Minor nit. This task should be below the task that was already here. The comments apply to the swift stoppage. And there should be a blank like between tasks.